Source/Charts/Components/MarkerImage.swift

```diff
@@ -102,7 +102,7 @@ open class MarkerImage: NSObject, IMarker
             height: size.height)
         
         NSUIGraphicsPushContext(context)
-        image!.draw(in: rect)
+        image?.draw(in: rect)
```

Comment:
Seems like there should be a `guard let image = image else { return }` at the beginning of the method. Drawing will do nothing if `image == nil`

Reply:
done